### PR TITLE
Fix FileHandleGenerator inside HiveConnector

### PR DESCRIPTION
**Pull request by majetideepak**
A recent refactor https://github.com/facebookincubator/velox/pull/7725 missed passing the connector config
to the FileHandleGenerator in the HiveConnector.

### CODE REVIEW

**Comment by majetideepak** [comment]
@mbasmanova can you please help review this? This fixes a regression.

File: velox/connectors/hive/storage_adapters/s3fs/tests/S3ReadTest.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox::exec::test;
+
+namespace facebook::velox {
+namespace {
+class S3ReadTest : public S3Test, public test::VectorTestBase {
+ public:
+  static constexpr char const* kMinioConnectionString{"127.0.0.1:6000"};
+  /// We use static initialization because we want a single version of the
```
Comment (mbasmanova) [inline velox/connectors/hive/storage_adapters/s3fs/tests/S3ReadTest.cpp:34]: nit: add empty line for readability

File: velox/connectors/hive/storage_adapters/s3fs/tests/S3ReadTest.cpp [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      "velox/connectors/hive/storage_adapters/s3fs/tests",
+      "../../../../../dwio/parquet/tests/examples/int.parquet");
+  const char* bucketName = "data";
+  const auto destinationFile = S3Test::localPath(bucketName) + "/int.parquet";
+  minioServer_->addBucket(bucketName);
+  std::ifstream src(sourceFile, std::ios::binary);
+  std::ofstream dest(destinationFile, std::ios::binary);
+  // Copy source file to destination bucket.
+  dest << src.rdbuf();
+  VELOX_CHECK_GT(dest.tellp(), 0, "Unable to copy from source {}", sourceFile);
```
Comment (mbasmanova) [inline velox/connectors/hive/storage_adapters/s3fs/tests/S3ReadTest.cpp:69]: ASSERT_GT?

**Review by mbasmanova — APPROVED** [review]
Thanks. Would you confirm that the new tests fails without the fix?

**Comment by mbasmanova** [comment]
@majetideepak CI is failing:

```
The following tests FAILED:
        289 - velox_s3read_test (Timeout)
```

**Comment by majetideepak** [comment]
> Thanks. Would you confirm that the new tests fails without the fix?

Yes, I confirmed the test fails without the fix.

**Comment by majetideepak** [comment]
@mbasmanova the test is now passing. Thanks!

**Pull request closed without merging**